src/models/User/createUser/validations/validateEmail/index.js

```diff
@@ -4,7 +4,7 @@ import validator from 'validator'
 export const INVALID_EMAIL_ERROR =
   'Your e-mail address does not seem to be valid.'
 export const EMAIL_ALREADY_IN_USE_ERROR =
-  'This e-mail address is already in use by another user.'
+  'This e-mail address is already used by another user.'
```

Comment:
The grammar warrior rises

Reply:
Is already in use is correct, but if it makes you happy 😄

Reply:
Then we should say `This e-mail address is already in use`

Reply:
🖕